tests/php/Integration/Command/Domain/DirectoryFinder/ComposerVendorDirectoriesFinderTest.php

```diff
@@ -4,18 +4,16 @@
 
 namespace PhelTest\Integration\Command\Domain\DirectoryFinder;
 
+use Gacela\Framework\Gacela;
 use Phel\Command\Domain\Finder\ComposerVendorDirectoriesFinder;
-use Phel\Config\PhelConfigException;
+use Phel\Phel;
 use PHPUnit\Framework\TestCase;
 
 final class ComposerVendorDirectoriesFinderTest extends TestCase
 {
-    public function test_exception_when_wrong_type(): void
```

Comment:
Wouldn't be interesting to test what happen if you return a different type in the `phel-config.php`? I mean, I would expect some exception? Or maybe is just ignoring it? 🤔

Reply:
A `RunetimeException` will be thrown, see https://github.com/gacela-project/gacela/blob/0.11.0/src/Framework/Config/ConfigReader/PhpConfigReader.php#L44

I was unsure whether or not to test this behaviour here, but I think we should not test framework code here. It is part of the `PhpConfigReader`'s conttract that it will throw that exception if it can't handle the config. Maybe you could add a custom exception in gacela and add it to the reader's interface, to make it more explicit.

Reply:
However, another question is how the `ComposerVendorDirectoriesFinder` should handle that exception. What does it mean if a vendor dependency has an invalid config file? Should the whole process terminate? Or should that vendor just be ignored? 

Depending on the answer(s) to that question I can add a try-catch and silently ignore that exception to avoid the whole loop to blow up.

Reply:
Good point. I think I like the idea of throwing a custom exception at phel level if the file throw a runtimeException